docs/02.API-REFERENCE.md

```diff
@@ -4679,3 +4753,170 @@ main (void)
 - [jerry_parse](#jerry_parse)
 - [jerry_run](#jerry_run)
 - [jerry_vm_exec_stop_callback_t](#jerry_vm_exec_stop_callback_t)
+
+
+# Array Buffer and Typed Array functions
+
+## jerry_arraybuffer_get_length
+
+**Summary**
+
+Get the length of the Array Buffer in bytes.
+
+**Prototype**
+
+```c
+jerry_length_t
+jerry_arraybuffer_get_length (const jerry_value_t value);
```

Comment:
There is a `jerry_get_array_length` function already. I am thinking whether this could be combined with it. Maybe just introducing a jerry_get_length() function which works with any special object (list in the comment what types it supports). But that could increase the binary size. I am unsure.

Reply:
The `jerry_arraybuffer_get_length` returns the `byteLength` property (which is specified ad construction time) and the arraybuffer does not have a `length` property. I'm not really sure we should remove this function, but we could change the name to have `byte_length` suffix?

Reply:
Maybe a name like: `jerry_get_arraybuffer_byte_length` ? so we'll have a common naming scheme for "getters".

Reply:
Hm, actually sounds good.